test/testrecipes/test-bicep-recipes/dapr-pubsub-broker.bicep

```diff
@@ -14,6 +14,7 @@ module redis '../../../test/testrecipes/modules/redis-selfhost.bicep' = {
   }
 }
 
+#disable-next-line BCP081
```

Comment:
What's this one?

Reply:
```
Resource type "dapr.io/Component@v1alpha1" does not have types available. Bicep is unable to validate resource properties prior to deployment, but this will not block the resource from being deployed.bicep(BCP081)
```

There's no linter rule alias for this so I used the number instead